### PR TITLE
Added link to docs on console help button fix #908

### LINKED ISSUE #908: link to docs on console help button
the help button on console currently links to GitHub, Discord and email. It should also link to docs.

image

### PR DESCRIPTION
**Pull request by dushyantRathore**
### Description

Added link to docs on console help button.

What component does this PR affect? 

- [ ] Server
- [x] Console
- [ ] CLI
- [ ] Docs
- [ ] Community Content
- [ ] Build System

Requires changes from other components? If yes, please mark the components:

- [ ] Server
- [ ] Console
- [ ] CLI
- [ ] Docs
- [ ] Community Content
- [ ] Build System

### Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update
- [ ] Community content

### Checklist:

- [ ] I have read the **[contributing guide](https://github.com/hasura/graphql-engine/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [ ] This change requires a change in the documentation. 
- [ ] I have updated the documentation accordingly.
- [ ] I have added required tests.

### CODE REVIEW

File: console/src/components/Main/Main.js [hunk 1]
```diff
@@ -306,6 +308,16 @@ class Main extends React.Component {
...
                       </a>
                     </li>
+                    <li className={'dropdown-item'}>
+                      <a href="https://docs.hasura.io/1.0/graphql/manual/index.html">
+                        <img
+                          className={'img-responsive'}
+                          src={docs}
+                          alt={'docs'}
+                        />
+                        <span>Docs</span>
```
Comment (shark-h) [inline console/src/components/Main/Main.js]: "Head to Docs" would look better.

File: console/src/components/Main/Main.js [hunk 2]
```diff
@@ -306,6 +308,16 @@ class Main extends React.Component {
                         <span>Reach out ({'support@hasura.io'})</span>
                       </a>
                     </li>
+                    <li className={'dropdown-item'}>
+                      <a href="docs.hasura.io ">
```
Comment (shark-h) [inline console/src/components/Main/Main.js]: "https://docs.hasura.io/"

**Comment by dushyantRathore** [comment]
@shark-h made the changes..kindly check

File: console/src/components/Main/Main.js (hunk 2 above)
Comment (dushyantRathore) [reply console/src/components/Main/Main.js]: @shark-h .. resolved the url

**Pull request closed without merging**